docs/source/conf.py

```diff
@@ -0,0 +1,340 @@
+# serve to show the default.
+
+import sys
+import os
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+# sys.path.insert(0, os.path.abspath('./'))
+
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ['sphinx.ext.intersphinx',
+              'sphinx.ext.viewcode',
+              'sphinx.ext.autodoc',
+              'sphinx.ext.autosectionlabel',
+              'autodocsumm',
+              # 'sphinx_automodapi.automodapi',
+              'sphinx.ext.napoleon',
+              # 'sphinxcontrib.autoprogram'
+              ]
+
+autosectionlabel_prefix_document = True
+autosectionlabel_maxdepth = 2
+autosummary_generate = True
+autosummary_generate_overwrite = True
+# numpydoc settings
+numpydoc_show_class_members = True
+numpydoc_show_inherited_class_members = True
+numpydoc_attributes_as_param_list = False
+numpydoc_class_members_toctree = False
+
+# Napoleon settings
+napoleon_google_docstring = True
+napoleon_numpy_docstring = True
+napoleon_include_init_with_doc = False
+napoleon_include_private_with_doc = False
+napoleon_include_special_with_doc = False
+napoleon_use_admonition_for_examples = True
+napoleon_use_admonition_for_notes = True
+napoleon_use_admonition_for_references = True
+napoleon_use_ivar = False
+napoleon_use_param = True
+napoleon_use_rtype = True
+
+# enable once new theme is final
+# html_style = "theme.css"
+
+# Add any paths that contain templates here, relative to this directory.
+# templates_path = ['_templates']
+
+# The suffix of source filenames.
+source_suffix = ['.rst', "md"]
+
+# The encoding of source files.
+# source_encoding = 'utf-8-sig'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = 'Pibooth'
+copyright = '2019-2021'
+github_doc_root = 'https://github.com/pibooth/pibooth/tree/master/docs/'
+issues_github_path = 'https://github.com/pibooth/pibooth/issues'
+
+latex_authors = ''''''
+
+# import
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+version = "2.0.3"
+
+if os.environ.get('READTHEDOCS') == 'True':
+    # Because Read The Docs modifies conf.py, versioneer gives a "dirty"
+    # version like "5.10.0+0.g28674b1.dirty" that is cleaned here.
+    version = version.partition('+0.g')[0]
+
+# The full version, including alpha/beta/rc tags.
+release = "2.0.3"
```

Comment:
import pibooth, then use pibooth.__version__ to get a dynamic update of the version

Reply:
OK